cli/azd/pkg/project/service_config.go

```diff
@@ -45,6 +45,8 @@ type ServiceConfig struct {
 	DotNetContainerApp *DotNetContainerAppOptions `yaml:"-,omitempty"`
 	// Custom configuration for the service target
 	Config map[string]any `yaml:"config,omitempty"`
+	// Environment variables for service
+	Env map[string]string `yaml:"env,omitempty"`
```

Comment:
Do we need to add environment variables here? IMU, it should be keep align with other services like PostgreSQL, currently, the PostgreSQL related environment variables are not displayed in azure.yaml.